app/views/includes/compareValueInput.scala.html

```diff
@@ -0,0 +1,41 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(
+    previousLabel: String,
+    newLabel: String,
+    newField: Field,
+    previousFieldName: String,
+    previousFieldValue: String
```

Comment:
Should this be a Moneyponds instead? This could be a runtime error if the previous value is malformed with a comma, and therefore fail for `toInt`

Reply:
I think this is a wider issue where we don't sanitize/validate our values from the form correctly before saving to the cache.